test/fixtures/recirculating_infusion_mash_systems.yml

```diff
@@ -4,8 +4,10 @@
 # model remove the '{}' from the fixture names and add the columns immediately
 # below each fixture, per the syntax in the comments below
 #
-one: {}
+rims_one:
+    id: 1
```

Comment:
I think you need some references to some fixtures representing valid subsystems (Thermostat, etc.). The RIMS controller tests are failing because the view is trying to get the pins of some Equipment that isn't defined (i.e. is nil).